src/metpy/plots/skewt.py

```diff
@@ -600,10 +627,14 @@ def plot_mixing_lines(self, w=None, p=None, **kwargs):
 
         # Set pressure range if necessary
         if p is None:
-            p = np.linspace(600, max(self.ax.get_ylim())) * units.mbar
+            # from highest pressure to 60% of highest pressure
+            pmax = max(self.ax.get_ylim())
+            p = np.linspace(0.6 * pmax, pmax) * self.ax.yaxis.units
+
+        # Dewpoint contours calculated and converted to xaxis units
+        td = dewpoint(vapor_pressure(p, w)).to(self.ax.xaxis.units)
```

Comment:
Is the manual conversion really necessary here given that we're setting axis units now?

Reply:
Apparently not, I have removed it in the next commit.